lib/bundles/inspec-compliance/api.rb

```diff
@@ -238,5 +243,11 @@ def self.server_version_from_config(config)
       return nil unless config['version'].is_a?(Hash)
       config['version']['version']
     end
+
+    def self.determine_server_type(url, insecure)
+      return 'automate' if Compliance::HTTP.get(url + '/compliance/version', nil, insecure).code == '401'
+      return 'compliance' if Compliance::HTTP.get(url + '/api/version', nil, insecure).code == '200'
+      raise CannotDetermineServerType
```

Comment:
You should also add an exception message when raising.

`raise CannotDetermineServerType, "Unable to determine if #{url} is a Compliance or Automate server"`

Reply:
Agreed. Fixed.